app/assets/stylesheets/mobile/topic-list.scss

```diff
@@ -460,6 +484,9 @@ td .main-link {
   }
 }
 .topic-list {
+  &-body {
```

Comment:
I think we should leave the border for now. This PR should only affect the drop down. I think we can address topic list changes in the on-going modernization of the topic list.

Reply:
I hear you but it takes away a lot of the lighter feel imo.